documentation/release_notes.markdown

```diff
@@ -10,6 +10,18 @@ canonical: "/puppetserver/latest/release_notes.html"
 [puppetserver.conf]: ./config_file_puppetserver.markdown
 [product.conf]: ./config_file_product.markdown
 
+## Puppet Server 6.11.1
+
+Released May 7 2020
+
+### Bug Fix
```

Comment:
seems like we use 'Resolved Issues' rather than 'Bug Fix' elsewhere in here.. Though I realize we're not positive that it's been resolved.. ¯\_(ツ)_/¯

Reply:
I'll update.